src/quo2/components/selectors/styles.cljs

```diff
@@ -1,130 +1,175 @@
+              :disabled {:on  (colors/alpha on-blur 0.3)
+                         :off (colors/theme-colors (colors/alpha off-blur-light 0.06)
+                                                   (colors/alpha off-blur-dark 0.12))}}}))
 
-(defn checkbox
-  [blurred-background? disabled? checked?]
-  {:flex             1
-   :border-radius    6
-   :border-width     (if checked? 0 1)
-   :background-color (cond
-                       checked?
-                       (get-color checked? disabled? blurred-background?)
-                       blurred-background?
-                       (colors/theme-colors
-                        colors/white-opa-5
-                        colors/white-opa-10)
-                       :else
-                       (colors/theme-colors
-                        colors/white
-                        colors/neutral-80-opa-40))
-   :border-color     (if checked?
-                       :none
-                       (get-color checked?
-                                  disabled?
-                                  blurred-background?))})
-
-(defn checkbox-toggle
-  [checked? disabled? blurred-background?]
-  {:flex             1
-   :border-radius    6
-   :border-width     (if @checked? 0 1)
-   :background-color (cond
-                       @checked?
-                       (get-color @checked? disabled? blurred-background?)
-                       blurred-background?
-                       (colors/theme-colors
-                        colors/white-opa-5
-                        colors/white-opa-10)
-                       :else
-                       (colors/theme-colors
-                        colors/white
-                        colors/neutral-80-opa-40))
-   :border-color     (if @checked?
-                       :none
-                       (get-color @checked? disabled? blurred-background?))})
+(def radio-background-color
+  (let [off      (colors/theme-colors colors/white-opa-40 colors/neutral-80-opa-40)
+        off-blur colors/white-opa-5]
+    {:normal {:enabled  off
+              :disabled (colors/alpha off 0.12)}
+     :blur   {:enabled  off-blur
+              :disabled (colors/theme-colors (colors/alpha off-blur 0.12)
+                                             (colors/alpha off-blur 0.015))}}))
 
-(defn radio
-  [checked? disabled? blurred-background?]
-  {:height           20
-   :width            20
-   :border-radius    20
-   :border-width     1
-   :border-color     (get-color checked?
-                                disabled?
-                                blurred-background?)
-   :background-color (when-not blurred-background?
-                       (colors/theme-colors colors/white
-                                            (colors/alpha colors/neutral-80
-                                                          0.4)))})
+(defn checkbox-background-color
+  [custom-color]
+  (let [on       (colors/custom-color-by-theme custom-color 50 60)
```

Comment:
very much a nit, but `checked/ unchecked` in place of `on`/`off`. Particularly for `on-blur` color which sounds like a callback

Reply:
Done e5ddbc394fa7a17fcf1458c5570a3ae90c2d5b26